test/unit/specs/components/common/SessionAccounts.spec.js

```diff
@@ -6,32 +6,37 @@ describe(`SessionAccounts`, () => {
   const localVue = createLocalVue()
   localVue.use(Vuelidate)
 
-  let wrapper
+  let wrapper, $store
 
   beforeEach(() => {
+    $store = {
+      state: {
+        accounts: [
+          {
+            name: "accountName",
```

Comment:
I would even change this to "Benji", or "Foobar's Account Name". Currently it looks like a variable, and this makes the test below harder to understand what is expected.